core/src/main/java/org/kiwix/kiwixmobile/core/reader/ZimFileReader.kt

```diff
@@ -217,15 +217,15 @@ class ZimFileReader constructor(
     title = this@ZimFileReader.title
     id = this@ZimFileReader.id
     size = "$fileSize"
-    favicon = this@ZimFileReader.favicon
+    favicon = this@ZimFileReader.favicon.toString()
     creator = this@ZimFileReader.creator
     publisher = this@ZimFileReader.publisher
     date = this@ZimFileReader.date
     description = this@ZimFileReader.description
     language = this@ZimFileReader.language
     articleCount = this@ZimFileReader.articleCount.toString()
     mediaCount = this@ZimFileReader.mediaCount.toString()
-    bookName = this@ZimFileReader.name
+    name = this@ZimFileReader.name
```

Comment:
`bookName` will give more context about the variable than `name`.

Reply:
done